frontend/src/FullPageRoutes/FAQ.tsx

```diff
@@ -0,0 +1,169 @@
+                    Penpals are anyone you are matched to.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>
+                    How many video message can be sent per day?
+                  </Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    One (1) video messages be per day can be sent.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>How long can the video be?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    The video can be between fifteen seconds to 10 minutes long.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+            </Grid>
+
+            <Grid item>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>What are the privacy settings?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    The privacy settings are public, private and hidden. Public
+                    allows for everyone to see your information. Private allows
+                    for just my penpals to see your information. Hidden allows
+                    for no one to see your information.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>Can I decline a penpal?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>Yes.</Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>
+                    Is there a maximum of penpals I can accept in a day?
+                  </Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>No.</Typography>
```

Comment:
"You can accept as many penpals as you want in a day. However, consider carefully how many different videos you want to record. We recommend you start out with a small number and add more penpals over time as you get more comfortable with the service."